log4j-layout-template-json-test/pom.xml

```diff
@@ -132,6 +132,13 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.elasticsearch.client</groupId>
+      <artifactId>elasticsearch-rest-client</artifactId>
```

Comment:
Can you implement the following changes, please?

1. Remove the `elasticsearch-java` dependency (We don't need this anymore, right?)
2. Place the `elasticsearch-rest-client` dependency adhering to the lexicographic order of artifact IDs
3. Replace hardcoded `<version>8...` with `<version>${elastic.version}</version>`